interfaces.go

```diff
@@ -6,17 +6,17 @@ import (
 	"net/http"
 )
 
-type DataWriter interface {
+type Response interface {
 	http.ResponseWriter
-	Data(interface{})
-	Status(int)
-	BlockData(string, *http.Request) (interface{}, bool)
+	Status(int) int
+	Done() bool
+	HandlePartial(string, *http.Request) interface{}
 	ResponseId() uint32
```

Comment:
Should be `ReponseID`; https://github.com/golang/lint/issues/124#issuecomment-102860034